environments/linux/docker/ubuntu.base.Dockerfile

```diff
@@ -81,21 +87,21 @@ ADD ./.upstream-tests/test                   /libcudacxx/test
 ADD ./.upstream-tests/utils                   /libcudacxx/utils
 
 # Install compiler and configure project
-RUN cmake -S /libcudacxx -B /build \
-        -G Ninja \
-        -DLIBCUDACXX_ENABLE_STATIC_LIBRARY=OFF \
-        -DLIBCUDACXX_ENABLE_LIBCUDACXX_TESTS=ON \
-        -DLIBCUDACXX_ENABLE_LIBCXX_TESTS=ON \
-        -DLIBCUDACXX_TEST_COMPILER_FLAGS="-allow-unsupported-compiler" \
-        -DLIBCUDACXX_TEST_WITH_NVRTC=${TEST_WITH_NVRTC} \
-        -DLIBCUDACXX_TEST_STANDARD_VER=c++${CXX_DIALECT} \
-        -DLIBCXX_ENABLE_FILESYSTEM=OFF \
-        -DCMAKE_CXX_COMPILER=${HOST_CXX} \
-        -DCMAKE_CUDA_COMPILER=${CUDACXX_PATH} \
-        -DCMAKE_CUDA_FLAGS="-allow-unsupported-compiler -std=c++14"
-
-RUN ninja -C /build libcudacxx_tu_tests && ninja -C /build clean
-RUN ninja -C /build cxx
+# RUN cmake -S /libcudacxx -B /build \
+#         -G Ninja \
+#         -DLIBCUDACXX_ENABLE_STATIC_LIBRARY=OFF \
+#         -DLIBCUDACXX_ENABLE_LIBCUDACXX_TESTS=ON \
+#         -DLIBCUDACXX_ENABLE_LIBCXX_TESTS=ON \
+#         -DLIBCUDACXX_TEST_COMPILER_FLAGS="-allow-unsupported-compiler" \
+#         -DLIBCUDACXX_TEST_WITH_NVRTC=${TEST_WITH_NVRTC} \
+#         -DLIBCUDACXX_TEST_STANDARD_VER=c++${CXX_DIALECT} \
+#         -DLIBCXX_ENABLE_FILESYSTEM=OFF \
+#         -DCMAKE_CXX_COMPILER=${HOST_CXX} \
+#         -DCMAKE_CUDA_COMPILER=${CUDACXX_PATH} \
+#         -DCMAKE_CUDA_FLAGS="-allow-unsupported-compiler -std=c++14"
+
+# RUN ninja -C /build libcudacxx_tu_tests && ninja -C /build clean
+# RUN ninja -C /build cxx
```

Comment:
Is that really intended to be commented out?

Reply:
No, it was for debugging something. I want to squash this MR once it's good to go.